doc-Managing_Providers/topics/Automation_Management_Providers.adoc

```diff
@@ -1,30 +1,42 @@
 [[automation_management_providers]]
 = Automation Management Providers
 
+[NOTE]
+====
+{product-title} ships with the Ansible role disabled. Navigate to menu:settings menu[Configuration > Settings] and select the desired server under *Zones*. Set the *Server Role* for *Embedded Ansible* to *On* to enable the role.
+====
 
 [[ansible-inside]]
 == Ansible
 
-Ansible integrates with {product-title} to provide automation solutions, using Playbooks, for Service, Policy and Alert actions. 
-Ansible Playbooks consist of series of _plays_ or tasks that define automation across a set of hosts,
+Ansible integrates with {product-title} to provide automation solutions, using playbooks, for Service, Policy and Alert actions. 
+Ansible playbooks consist of series of _plays_ or tasks that define automation across a set of hosts,
 known as the inventory. 
 
-Ranging from simple to complex tasks, Ansible Playbooks can support cloud management:
+Ranging from simple to complex tasks, Ansible playbooks can support cloud management:
 
-* *Services* - allow a Playbook to back a {product-title_short} service catalog item.
-* *Control Actions* - {product-title_short} policies can execute Playbooks as actions based on events from providers.
-* *Control Alerts* - set a Playbook to launch prompted by a {product-title_short} alert.
+* *Services* - allow a playbook to back a {product-title_short} service catalog item.
+* *Control Actions* - {product-title_short} policies can execute playbooks as actions based on events from providers.
+* *Control Alerts* - set a playbook to launch prompted by a {product-title_short} alert.
 
 Ansible is built into {product-title_short} so there is nothing to install. The basic workflow when using Ansible in {product-title} is as follows:
 
-* Add a source control repository that contains your Playbooks.
+* Add a source control repository that contains your playbooks.
 * Establish credentials with your inventory. 
-* Back your services, alerts and policies using available Playbooks. 
+* Back your services, alerts and policies using available playbooks. 
+
+[[verifying-embedded-ansible-worker-state]]
+=== Verifying the Embedded Ansible Worker State
+Verify that the Embedded Ansible worker has started to utilize its features. 
```

Comment:
I'm not sure if Embedded Ansible qualifies as a proper name and should be capitalized.